.gitignore

```diff
@@ -9,6 +9,7 @@ example/node_modules
 build
 dist
 .rpt2_cache
+storybook-static
```

Comment:
intentional?

Reply:
added bc this was generated when i built, but doesn't need to be in this PR 👍